tailwind.config.js

```diff
@@ -34,10 +34,6 @@ module.exports = {
       backgroundSize: {
         50: "50px 50px",
       },
-      borderWidth: {
-        1: "1px",
-        16: "16px",
-      },
```

Comment:
border-1 was just an experiment, and border-16 was the previous style of the site border before I abandoned the truly retro click-around-to-various-pages layout